crates/primitives/src/account.rs

```diff
@@ -63,16 +63,10 @@ impl Bytecode {
     }
 
     /// Create new bytecode from raw bytes and its hash.
-    pub fn new_raw_with_hash(bytes: Bytes, code_hash: H256) -> Self {
-        let revm_bytecode = unsafe { RevmBytecode::new_raw_with_hash(bytes, code_hash) };
+    pub fn new_raw_with_hash(bytes: Bytes) -> Self {
+        let revm_bytecode = unsafe { RevmBytecode::new_raw_with_hash(bytes) };
         Self(revm_bytecode)
     }
```

Comment:
if this is no longer taking a hash, how is it _new_raw_with_hash?

Reply:
Will be removed: https://github.com/bluealloy/revm/issues/674